tests/spec_decode/e2e/test_mlp_correctness.py

```diff
@@ -203,7 +203,7 @@ def test_mlp_e2e_acceptance_rate(vllm_runner, common_llm_kwargs,
 @pytest.mark.parametrize("test_llm_kwargs", [{"seed": 5}])
 @pytest.mark.parametrize("output_len", [64])
 @pytest.mark.parametrize("batch_size", [1, 32])
-@pytest.mark.parametrize("temperature", [0.1, 1.0])
+@pytest.mark.parametrize("temperature", [0.6, 1.0, 1.2])
```

Comment:
nit - wondering if it would be ok to run it only for temperature 1.0?

Reply:
I updated it ! :)

Reply:
Hi I think the failures in distributed-tests-a100 may not be related to this pr (I see errors related CustomAllreduce in the failure logs). I see this test fail in some other prs as well. The other 2 failures seem related to timeouts.